### DISCUSSION TITLE
Is there any method to check if an object has member?

### DISCUSSION THREAD

**Discussion by tuongbuimtt** [yosh-matsuda/cpp-yyjson#13] [Q&A] [answered]
I've tried, but it doesn't seem be the correct method:
```
    object obj;
    obj.emplace("ABC", 123);
    if (obj["abc"].is_null())
      cout<< "object has no member named abc" << endl;
     cout << obj.write() << endl;
```
Of course, the condition is true, and that will print the string, but after call the is_null(), the obj has a member "abc"
```
object has no member named abc
{"ABC":123,"abc":null}
```

**Comment by yosh-matsuda** [comment, marked as answer]
@tuongbuimtt Check the key of each member using the object iterator or use the `contains` and `find` methods added in v0.4.0. These behave like STL.

**Reply by tuongbuimtt** [reply]
Thanks for your support so much.
